src/utils/fileUtils.test.tsx

```
@@ -107,10 +107,57 @@ describe('fetchFileAndExtractText', () => {
+    splitSpy.mockRestore()
     // restore real File
     // @ts-ignore
     globalThis.File = OriginalFile
   })
+
+  it('works even when File.prototype.arrayBuffer is initially missing', async () => {
+      // temporarily delete the method
+      const saved = File.prototype.arrayBuffer
+      // @ts-ignore
+      delete File.prototype.arrayBuffer
+   
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)